web-api/terraform/template/cognito.tf

```diff
@@ -1,41 +1,32 @@
 resource "aws_cognito_user_pool" "pool" {
   name = "efcms-${var.environment}"
 
-  auto_verified_attributes = ["email"]
-
   username_attributes = ["email"]
 
   verification_message_template {
-    default_email_option  = "CONFIRM_WITH_LINK"
-    email_message_by_link = <<EMAILMESSAGE
-    <body>
-      <p>Welcome to DAWSON!</p>
-
-      <p>Your account with DAWSON has been created. Use the button below to verify your email address.</p>
-
-      <span>{##<button style="font-family: Source Sans Pro Web,Helvetica Neue,Helvetica,Roboto,Arial,sans-serif;
-          font-size: 1.06rem;
-          line-height: .9;
-          color: #fff;
-          background-color: #005ea2;
-          border: 0;
-          border-radius: 0.25rem;
-          cursor: pointer;
-          display: inline-block;
-          margin-right: 0.5rem;
-          padding: .75rem 2.25rem;
-          text-align: center;
-          text-decoration: none;">Verify Email</button>##}
-      </span>
-
-      <p>If you did not create an account with DAWSON, contact <a href="mailto:dawson.support@ustaxcourt.gov">dawson.support@ustaxcourt.gov</a>.</p>
-
-      <hr>
-
-      <p>This is an automated email. We are unable to respond to any messages to this email address.</p>
-    </body>
+    default_email_option = "CONFIRM_WITH_CODE"
+    email_message        = <<EMAILMESSAGE
+    <div>
+    <div>
+      Hello DAWSON user, 
```

Comment:
😂

I realize this is worded this way because we don't know their name yet, but "Hello DAWSON user" amuses me.